File: transpilers/babel/prog_sample_programs/3.js
// Exercising the construct: guard clause with early return inside arrow function

let x = 5;

const guardClauseExample = () => {
    if(x < 0)
        return "Negative number";
    if(x > 10)
        return "Number larger than 10";
    return x;
}

let func1 = () => {
    return guardClauseExample();
}

let main = () => {
    let res = func1();
    console.log(res);
}

main();

File: transpilers/tinygo/prog_sample_programs/3.go
// This program is exercising the construct of a for loop with early exit via goto in Go

package main

import "fmt"

func func1() int {
	var res int
	for i := 0; i <= 10; i++ {
		if i > 7 {
			goto Exit
		}
		res += i
	}

Exit:
	return res
}

